resources/lib/cabertoss.py

```
@@ -88,17 +88,17 @@ def gather_log_files():
-def copy_log_files(log_files: []):
+def copy_log_files(log_files: List) -> bool:
     """
     Actually copy the log files to the path in the addon settings
 
-    @param log_files: [] list of log files to copy
-    @return: None
+    @param log_files: List list of log files to copy
+    @return bool: indicating success or failure
     """
     if not log_files:
         Logger.error(LANGUAGE(32025))
```

Ruff (0.13.1)

`LANGUAGE` may be undefined, or defined from star imports

(F405)